cli/azd/CHANGELOG.md

```diff
@@ -2,6 +2,52 @@
 
 ## Upcoming (Unreleased)
 
+### Important Notes
```

Comment:
IMHO, this should be part of `breaking changes`.
the Breaking changes are the most important notes from the changelog.  

By having `Important Notes`, I would wonder:
- Is it a breaking change?  is it a new feature?  are other sections not important notes? :)
- Are we deviating from the standard Azure SDK's release changelog sections?  
- Usually, within the Azure SDK, there is a blog post which highlights the most important changes on the top, by just elevating one of the entries of the changelog.  Should we have a note like the one in this PR to be on the top of some of the @savannahostrowski 's blog posts instead? (while keeping a single entry here within the breaking changes?)

Reply:
We will include this update in the January release post. @ellismg is there an issue associated with this PR? Looking to tag it as "newsworthy" so we don't forget to highlight it in the next monthly update.

Reply:
@vhvb1989 I had moved this up to the "top" of the release notes because I wanted it to the first thing people saw.  I actually still have a note about this in the breaking changes section (if you expand the context of the diff, I think you'll see it, but here's the content):

- [[#1022]](https://github.com/Azure/azure-dev/pull/1022) `azd` no longer uses the `az` CLI to authenticate with Azure by default. You will need to run `azd login` after upgrading. You may run `azd config set auth.useAzCliAuth true` to restore the old behavior of using `az` for authentication.

My general thinking here was that we'd use the top of the changelog to really highlight things, and then have the list of changes come later, but I see your point about stuff being regular being useful I'll move the prose down into that section.